api/net/Areas/Helpers/ReportHelper.cs

```diff
@@ -91,7 +91,7 @@ public async Task<IEnumerable<ContentModel>> FindContentAsync(JsonDocument filte
         var instance = _reportService.GetCurrentReportInstance(reportId, ownerId, true);
         var sections = report.Sections.ToDictionary(s => s.Name, s =>
         {
-            var content = instance?.ContentManyToMany.Where(c => c.Content != null && c.SectionName == s.Name) ?? Array.Empty<Entities.ReportInstanceContent>();
+            var content = instance?.ContentManyToMany.Where(c => c.Content != null && c.SectionName == s.Name).ToArray() ?? Array.Empty<Entities.ReportInstanceContent>();
```

Comment:
I've added `.ToArray()` in several places in hope it would catch the odd phantom behaviour we're seeing with Scott's account regularly missing content.  It's a total guess and I still need to debug.